src/models/DefaultRawDataType.ts

```diff
@@ -0,0 +1,7 @@
+
+/**
+ * The default type for "rawData" field of type Result.
```

Comment:
nit: can we link to `Result` or `rawData` here?

Reply:
nope it would say something like `Unable to resolve reference "Result": The member reference "Result" was not found` since we don't export Result from component lib

Reply:
ah right